yarn-project/noir-compiler/src/compile/noir/dependencies/dependency-resolver.ts

```diff
@@ -2,6 +2,16 @@ import { NoirDependencyConfig } from '@aztec/foundation/noir';
 
 import { NoirPackage } from '../package.js';
 
+/**
+ * A Noir dependency
+ */
+export type Dependency = {
```

Comment:
`Dependency` is a bit generic, would it make sense to name it `NoirDependency`?

Reply:
I renamed it to `NoirDependency` and renamed the resolver to `NoirDependencyResolver` to keep consistent